optuna_dashboard/ts/types/index.d.ts

```diff
@@ -203,6 +204,8 @@ type StudyDetail = {
   is_preferential: boolean
   objective_names?: string[]
   form_widgets?: FormWidgets
+  feedback_component_type: FeedbackComponentType
+  feedback_artifact_key?: string
```

Comment:
I think these attributes do not match with the API response. Does it work?

https://github.com/optuna/optuna-dashboard/blob/76356eb233a754a881ae2fcda11e1c6958388d01/optuna_dashboard/_serializer.py#L169-L171